src/pages/Page.styled.ts

```diff
@@ -0,0 +1,5 @@
+import { Box, styled } from '@mui/material';
+
+export const StyledBox = styled(Box)(() => {
+  return { alignItems: 'center', display: 'flex', justifyContent: 'center' };
+});
```

Comment:
Each page should have its directory. Check the docs
https://github.com/dorm-social-room-manager/dsrm-frontend/wiki/Project-structure#srcpages

Reply:
changed in the latest commit.